### PR TITLE
Dilepton

### PR DESCRIPTION
**Pull request by swertz**
Added 
- selected muon/electron indices (loose, tight, isolated, "selected" = tight+isolated+cuts)
- leading lepton pairs indices (only the one leading pair)
- exclusive dilepton categories as a function of the leading lepton pairs
- Z-peak exluding categories in ee/mumu final states (we might want to add as extra criterium that none of the lepton pairs should be in the Z peak, even the non-leading ones)

### CODE REVIEW

File: test/TTConfigurationData.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            pt_cut = cms.untracked.double(30)
+            )
+        ),
+
+    tt = cms.PSet(
+        type = cms.string('tt_analyzer'),
+        prefix = cms.string('tt_'),
+        enable = cms.bool(True),
+        parameters = cms.PSet(
+            electronIsoCut = cms.untracked.double(11),
```
Comment (blinkseb) [inline test/TTConfigurationData.py]: Not sure if it's intended to be super loose or it's just a mistake (isolation is stored relative)

Comment (swertz) [inline test/TTConfigurationData.py]: It's a mistake, it should be .11... Thanks for spotting! That explains the e/mu/e event I'd seen.

File: test/TTConfigurationData.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            electronPtCut = cms.untracked.double(20),
+            electronEtaCut = cms.untracked.double(2.4),
+            muonIsoCut = cms.untracked.double(12),
```
Comment (swertz) [inline test/TTConfigurationData.py]: Should be .12 (note to myself)

File: test/TTConfigurationData.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            mll_cut = cms.untracked.double(20),
+            mll_cut_low = cms.untracked.double(86),
+            mll_cut_high = cms.untracked.double(116)
+            ),
+        )
+
+    ), redoJEC=True
+    )
+
+Framework.schedule(process, ['dilepton', 'tt'])
```
Comment (blinkseb) [inline test/TTConfigurationData.py]: `bTags*` analyzers are missing here.

**Pull request merged**